deluge/plugins/AutoAdd/deluge/plugins/autoadd/gtkui.py

```diff
@@ -299,43 +299,42 @@ def on_cancel(self, event=None):
     def generate_opts(self):
         # generate options dict based on gtk objects
         options = {}
-        options['enabled'] = self.glade.get_widget('enabled').get_active()
+        options['enabled'] = self.main_builder.get_object('enabled').get_active()
         if client.is_localhost():
-            options['path'] = self.glade.get_widget('path_chooser').get_filename()
-            options['download_location'] = self.glade.get_widget(
+            options['path'] = self.main_builder.get_object('path_chooser').get_filename()
+            options['download_location'] = self.main_builder.get_object(
                 'download_location_chooser').get_filename()
-            options['move_completed_path'] = self.glade.get_widget(
+            options['move_completed_path'] = self.main_builder.get_object(
                 'move_completed_path_chooser').get_filename()
-            options['copy_torrent'] = self.glade.get_widget(
+            options['copy_torrent'] = self.main_builder.get_object(
                 'copy_torrent_chooser').get_filename()
         else:
-            options['path'] = self.glade.get_widget('path_entry').get_text()
-            options['download_location'] = self.glade.get_widget(
+            options['path'] = self.main_builder.get_object('path_entry').get_text()
+            options['download_location'] = self.main_builder.get_object(
                 'download_location_entry').get_text()
-            options['move_completed_path'] = self.glade.get_widget(
+            options['move_completed_path'] = self.main_builder.get_object(
                 'move_completed_path_entry').get_text()
-            options['copy_torrent'] = self.glade.get_widget(
+            options['copy_torrent'] = self.main_builder.get_object(
                 'copy_torrent_entry').get_text()
 
-        options['label'] = self.glade.get_widget('label').child.get_text().lower()
```

Comment:
should be more like 
options['label'] = self.main_builder.get_object('label').child.get_text().lower() 
however you get
#  AttributeError: 'ComboBox' object has no attribute 'child'

deffinatly not correct and needs a TODO